package.json

```diff
@@ -26,13 +27,19 @@
     "babel-preset-stage-0": "^6.22.0",
     "copy-webpack-plugin": "^4.0.1",
     "css-loader": "^0.27.3",
+    "eslint": "^3.19.0",
+    "eslint-config-airbnb": "^14.1.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^4.0.0",
+    "eslint-plugin-react": "^6.10.3",
     "json-loader": "^0.5.4",
-    "react": "^15.4.1",
-    "react-dom": "^15.4.1",
     "webpack": "^2.2.1",
     "webpack-dev-server": "^2.3.0"
   },
   "dependencies": {
+    "prop-types": "^15.5.8",
+    "react": "^15.4.1",
+    "react-dom": "^15.4.1",
```

Comment:
And yes @mamodom, it's quite difficult because of the naming but that's part of the linting process.